src/script/script.cpp

```diff
@@ -326,3 +329,11 @@ bool GetScriptOp(CScriptBase::const_iterator& pc, CScriptBase::const_iterator en
     opcodeRet = static_cast<opcodetype>(opcode);
     return true;
 }
+
+bool IsOpSuccess(const opcodetype& opcode)
+{
+    return (opcode == 0x50 || opcode == 0x62 || opcode == 0x89 ||
```

Comment:
Why not check against opcode defines?

Reply:
Because they're all "OP_SUCCESS"?  They aren't other opcodes right now. Some of them share values with non-tapscript opcodes, but the usage is different.

Reply:
I was expecting something along the lines of OP_SUCCESS80, ..., OP_SUCCESS254.
OP_SUCCESSx mapping explicitly shows the shared opcodes?

Reply:
I considered that, but I don't think the amount of code needed is worth it, or would be more readable (if there isn't a normative IsOpSuccess function, you'd be adding 87 constants to the enum, and instead writing "opcode == OP_SUCCESS80 || opcode == OP_SUCCESS98 || opcode == OP_SUCCESS126 || ..." in `ExecuteWitnessProgram` directly.

Reply:
Ah yes, good point about 87 constants, thanks for the clarification.